src/components/Header/header.scss

```diff
@@ -1,8 +1,26 @@
+@import '~bootstrap/scss/_functions';
+@import '~bootstrap/scss/_variables';
+@import '~bootstrap/scss/mixins/_breakpoints';
+@import '../standard-variables.scss';
+
 .header{
     background-position: center;
     background-repeat: no-repeat;
     background-size: cover;
     min-width: 100%;
     min-height: 100%;
     height:100vh;
+    h1{
+        font-family: $font-stack;
+    }
+}
+@include media-breakpoint-down(sm) { 
+    .header{
+        min-height: 9.5rem;
```

Comment:
Why are you using rem and % for SASS. Since u are implementing SASS, you should be using its features like mixins, conditions. Most importantly, use variables for colors and sizes. Give them a generic name.